frontend/src/components/Help.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import '../styles/About.css';
+
+
+class Help extends React.Component {
+  render () {
+    return (
+      <div className="about-modal-container">
+        <h1 className="about-logo">SPLAT CONTROLS</h1>
```

Comment:
This header gets squashed onto one line at small window widths -- make h1 wrap onto two lines if need be